src/ucs/config/global_opts.c

```diff
@@ -191,6 +191,39 @@ static ucs_config_field_t ucs_global_opts_table[] = {
   "The list order decides the priority of the providers.",
   ucs_offsetof(ucs_global_opts_t, topo_prio), UCS_CONFIG_TYPE_STRING_ARRAY},
 
+  {"PCI_ROOT_LAT", "300ns",
+   "Estimated latency between devices with the same PCI domain and bus number",
+   ucs_offsetof(ucs_global_opts_t, pci_root_lat), UCS_CONFIG_TYPE_TIME},
+
+  {"PCI_ROOT_BW_BASE", "19200MBs",
+   "Divisible part of the estimated bandwidth between devices with the same\n"
+   "PCI domain and bus number. The path between devices is the divisor of the\n"
+   "estimation.",
+   ucs_offsetof(ucs_global_opts_t, pci_root_bw_base), UCS_CONFIG_TYPE_BW},
+
+  {"PCI_ROOT_BW_MAX", "3500MBs",
+   "Maximum estimated bandwidth between devices with the same PCI domain and\n"
+   "bus number",
```

Comment:
I think current descriptions of `PCI_ROOT_BW_BASE` and `PCI_ROOT_BW_MAX` may be little confusing for person who will read that without knowledge of the formula in `topo.c` since they kind of intersecting.

I want to propose the following (maybe the description still isn't perfect, but that's more about renaming of variable):

```suggestion
  {"PCI_ROOT_BW_UPPER_BOUND", "3500MBs",
   "Upper bound of estimated bandwidth between devices with the same PCI domain and\n"
   "bus number.  Final PCI root bandwidth estimations are capped by this value.",
```

Reply:
I've updated the description. I think the name `PCI_ROOT_BW_MAX` fits well. Because this is a maximum value. So the name match the description, and the purpose. I changed the other variable. I guess it was more unclear than this one. Please check the new name and description.

Reply:
I left the calculation unchanged. The logic was added by this PR https://github.com/openucx/ucx/pull/7390.